django_x509/base/admin.py

```diff
@@ -64,6 +64,8 @@ def get_fields(self, request, obj=None):
         # edit
         if obj and 'extensions' in fields:
             fields.remove('extensions')
+        if obj and 'passphrase' in fields:
+            fields.remove('passphrase')
```

Comment:
I changed the previous lines on the master branch in https://github.com/openwisp/django-x509/commit/ade4ac3992341e1599cf0bffa75b1aaac2fbb68a

Please rebase your dev branch on master (eg: `git rebase origin/master`), after the rebase this section should now be:
```python
if obj:
    fields = fields[:]  # make copy
    fields.remove('extensions')
    fields.remove('passphrase')
```